dex/farm-with-locked-rewards/src/lib.rs

```diff
@@ -122,12 +116,7 @@ pub trait Farm:
         &self,
         opt_orig_caller: OptionalValue<ManagedAddress>,
     ) -> ClaimRewardsResultType<Self::Api> {
```

Comment:
Should we rename the endpoint to `claimBaseRewards`? To properly describe what happens inside the function? The same for the farm and farm-staking contracts.

Reply:
Nah, people hate endpoint renames, especially the microservice guys.